recipe/conda_build_config.yaml

```diff
@@ -8,6 +8,9 @@ major_minor_ver:
   - 11.2
   - 11.2.1
   - 11.2.2
+  - 11.3
+  - 11.3.1
```

Comment:
Why do we need both?

Reply:
Probably not needed, but no harm in adding if users want it. We seem to very rarely update this feedstock (only during releases of new versions really). Though if we are concerned about this job remaining in the build matrix, we could always drop this right after the package is published. WDYT?

Reply:
I don't think there's any point adding an older version when there's a new one. 

> no harm in adding if users want it. 

Sure, but let's wait until a user asks for it.

Reply:
@jaimergp would you be ok with dropping `11.3` and only keeping `11.3.1`?

Reply:
We could probably drop 11.2 and 11.2.1 while we're at it as well

Reply:
Yes, I think I only added both because they kept appearing as the PR stayed open. I guess keeping the last available for each `major.minor` is fair.

Reply:
I will keep the metadata in the meta.yaml dict though for reference / provenance.